build/media-suite_compile.sh

```diff
@@ -1220,6 +1221,28 @@ if [[ $mp4box = "y" ]] && do_vcs "https://github.com/gpac/gpac.git"; then
     do_checkIfExist
 fi
 
+_check=(SvtHevcEnc.pc libSvtHevcEnc.dll.a svt-hevc/EbApi.h
+    bin-video/Svt-Hevc/{libSvtHevcEnc.dll,SvtHevcEncApp.exe})
+if [[ $svthevc = "y" ]] && do_vcs "https://github.com/OpenVisualCloud/SVT-HEVC.git"; then
+    do_uninstall bin-video/SVT-Hevc/libSvtHevcEnc.dll.a "${_check[@]}"
+    ORIGINAL_PKG_CFG=$PKG_CONFIG PKG_CONFIG="$LOCALDESTDIR/bin/ab-pkg-config-static.bat"
```

Comment:
Any special reason for this? You can override PKG_CONFIG in the cmake call as well without having to change the global variable.

Also, why not `do_cmake` below? Is there any broken option in it that shouldn't be called specifically for svthevc?

Reply:
At the time, I wasn't exactly sure what used the PKG_CONFIG variable, will change.

I cannot use the `-DUNIX=ON` for do_cmake since in the cmakelists for SVT-HEVC, I have some of the things filtered from APPLE, UNIX, else and some from UNIX else

Reply:
I might just bisect the do_basecmake from the other pr and add that to a separate pr so I can use that instead